Remove autoprefixer as it's already included (#694)
withastro/docs: src/pages/en/guides/styling.md

## PostCSS

Astro comes with PostCSS included as part of [Vite](https://vitejs.dev/guide/features.html#postcss). To configure PostCSS for your project, create a `postcss.config.js` file in the project root. You can import plugins using `require()` (don't forget to install them: `npm i -D postcss-preset-env`).

```js
// ./postcss.config.js

module.exports = {
  plugins: [

>     require('postcss-preset-env'),

Reviewer: ```suggestion
    require('autoprefixer'),
    require('cssnano'),
```

## PostCSS

> Astro comes with PostCSS included as part of [Vite](https://vitejs.dev/guide/features.html#postcss). To configure PostCSS for your project, create a `postcss.config.js` file in the project root. You can import plugins using `require()` (don't forget to install them: `npm i -D postcss-preset-env`).

Reviewer: ```suggestion
Astro comes with PostCSS included as part of [Vite](https://vitejs.dev/guide/features.html#postcss). To configure PostCSS for your project, create a `postcss.config.js` file in the project root. You can import plugins using `require()` after installing them (for example `npm i autoprefixer`).
```

module.exports = {
  plugins: [
    require('postcss-preset-env'),

>   ],

Reviewer: How about we show use of `cssnano` here instead? It’s [the second most popular plugin](https://www.npmjs.com/search?q=keywords%3Apostcss-plugin&ranking=popularity) after `autoprefixer` according to npm.

```suggestion
    require('cssnano'),
  ],
```

Reviewer: Any thoughts on this @inwardmovement?

Author: Yes I think it would be more complementary to `postcss-preset-env` than `autoprefixer` (and `autoprefixer` is redundant with `postcss-preset-env`).

Or why not show only `autoprefixer`? It's pretty familiar to everyone I assume so it acts as a kind of landmark.

But my proposal was more to not show redundant plugins, as it's not really "real-world-ish", than to show plugin X or Y.

Reviewer: Sounds like a plan!
